products/liquibase.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Liquibase
+category: framework
+tags: app library amazon apache google ibm microsoft oracle sap vmware
+iconSlug: liquibase
+permalink: /liquibase
+versionCommand: liquibase --version
+changelogTemplate: https://docs.liquibase.com/start/release-notes/liquibase-release-notes/liquibase-__LATEST__.html
+
+identifiers:
+-   repology: liquibase
+
+auto:
+  methods:
+  -   git: https://github.com/liquibase/liquibase.git
+
+# eol(x) = releaseDate(x+1)
+releases:
```

Comment:
[4.32 has been released](https://github.com/liquibase/liquibase/releases/tag/v4.32.0) not long ago, can you add it ?